lib/scenic/statements.rb

```diff
@@ -103,7 +112,16 @@ def update_view(name, version: nil, sql_definition: nil, revert_to_version: nil,
         )
       end
 
-      sql_definition ||= definition(name, version)
+      if defined_as.present? && sql_definition.present?
```

Comment:
repetitive code: should this be a shared method? or a configuration object? (Not that you need to do that for someone else's code base, but just a thought.)

Reply:
we also use "present" here rather than "blank" above: should these match?